symengine/symengine_casts.h

```diff
@@ -55,8 +55,9 @@ inline To implicit_cast(const From &f)
 // You should design the code some other way not to need this.
 
 template <typename To, typename From> // use like this: down_cast<T*>(foo).
-inline To down_cast(From *f)          // only accept pointers.
+inline To down_cast(From *f)
 {
+    // Only accept pointers.
```

Comment:
@isuruf I must've accidentally made the change. I'll correct it (anyways the build needs to be re-run as Travis had an issue with OSX related jobs previously).